Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+	<head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        
+		<title>GirlDevelopIt</title>
+		<link href = "css/styles.css" rel = "stylesheet" type ="text/css">
+        
+            <!-- Bootstrap -->
+        <link rel="stylesheet" href="Bootstrap/css/bootstrap.min.css" />
+        <link rel="stylesheet" href="Bootstrap/css/bootstrap.css" />
+        <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+        <!-- Include all compiled plugins (below), or include individual files as needed -->
+        <script src="Bootstrap/js/bootstrap.min.js"></script>
+	</head
+	<body>
+        
```

Comment:
Add a bootstrap container here, it will help to give a fixed layout working with in all devices
https://getbootstrap.com/docs/4.0/layout/overview/#containers